test/config_parser_SUITE_data/mongooseim-pgsql.cfg

```diff
@@ -0,0 +1,939 @@
+%%%
```

Comment:
Does this file need all the comments explaining things? Nothing important, but it might just reduce repo size a few bytes for duplicated comments coming from the original config file 🤔

Reply:
Right now it's just a copy-paste of the original config file and I didn't want to change it to much, making it more difficult to compare with the original file. That's also how the rest of the tests for `ejabberd_config` work.